src/ui/__tests__/ui_i18n_keys.js

```diff
@@ -0,0 +1,50 @@
+import expect from 'expect.js';
```

Comment:
I think @epixa preferred to run these verifications at build time, whereas this will only run them with the unit tests. See my other comment on the i18n_verify_keys module for more discussion on this topic.